bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -117,13 +119,22 @@ spec:
         displayName: Shared Parameters
         path: template.spec.sharedparameters
       version: v1alpha1
-  description: "Fence Agents Remediation Operator (FAR)\n\nThe operator will remediate/fence
-    a node when a FenceAgentsRemediation (far) custom resource exists with the node's
-    name.\nFAR uses an existing set of upstream fencing agents for environments with
-    a \ntraditional API end-point (e.g., IPMI) for power cycling cluster nodes.\n\nThe
-    operator is recommended to be used with Node Healthcheck Operator for easier and
-    smoother experience,\nbut it can also work as a standalonde remediator for the
-    more advanced user.\n"
+  description: |
+    Fence Agents Remediation (*FAR*) is a Kubernetes operator that *fence* and remediate unhealthy to healthy nodes.
+    Using a traditional Application Programming Interface (API) FAR runs uses well-known fence agents to remediate a node from an unhealthy state by power cycling the node,
+    and afterwards it delets the node's resources.
+    By doing so, FAR can minimize downtime for stateful applications, restores compute capacity if transient failures occur, and increases the availability of workloads.
```

Comment:
I think this is an incorrect. The sentence begins with "FAR can..." so all the other verbs shouldn't use the third form present tense (FAR can minimize, can restore, can increase).
Maybe, you could just remove the first "can" and begin with "FAR minimizes..."?

Reply:
>The sentence begins with "FAR can..." so all the other verbs shouldn't use the third form present tense (FAR can minimize, can restore, can increase).

The `can` is intended as a best effort from FAR, I will just use V1 of the verbs.